vocode/streaming/telephony/server/base.py

```diff
@@ -138,17 +150,64 @@ def route(
                 base_url=self.base_url, call_id=conversation_id
             )
 
-        return route
+        def vonage_route(
+            vonage_config: VonageConfig, vonage_answer_request: VonageAnswerRequest
+        ):
+            call_config = VonageCallConfig(
+                transcriber_config=inbound_call_config.transcriber_config
+                or DeepgramTranscriberConfig(
```

Comment:
seen this a couple times (in constructor too) – should we define a helper?

Reply:
good call